### PR TITLE
Bump vsnp_build_tables requirements

### PR DESCRIPTION
**Pull request by gregvonkuster**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by gregvonkuster** [comment]
@bernt-matthias this one needs a review before I can merge it.  Thanks!

**Comment by bgruening** [comment]
@gregvonkuster can you rebase this PR, we hav a merge conflict as it seems.

File: tools/vsnp/macros.xml [hunk 1]
```diff
@@ -2,6 +2,21 @@
 <macros>
     <token name="@WRAPPER_VERSION@">1.0</token>
     <token name="@PROFILE@">19.09</token>
```
Comment (bernt-matthias) [inline tools/vsnp/macros.xml:4]: Please increase the profle to 20.05 or at least 20.01

**Comment by gregvonkuster** [comment]
Closing this broken pr.   I'll try to fix things in a new one.

**Pull request closed without merging**